libs9s/s9soptions.cpp

```diff
@@ -6626,7 +6616,6 @@ S9sOptions::printHelpBackup()
 "  --parallellism=N           Number of threads used while creating backup.\n"
 "  --pitr-compatible          Create backup compatible with PITR.\n"
 "  --pitr-stop-time           Do point in time recovery with specified time.\n"
-"  --psql-immediate           PostgreSql restore basebackup, don't apply WAL.\n"
```

Comment:
Does it also apply to "pgbackrest"?

Reply:
Yes. This option only had sense if we use it for PgBackRest, pg_basebackup and whatever else later. If we drop it for one, we better drop for all.